sdk/identity/identity/test/node/msalNodeTestSetup.ts

```diff
@@ -46,9 +44,6 @@ export async function msalNodeTestSetup(
 
   const sandbox = createSandbox();
 
-  const stub = sandbox.stub(util, "randomUUID");
```

Comment:
This is just a better world to live in, no weird wrapping and no concerns with immutable modules